sos/policies/distros/redhat.py

```diff
@@ -624,7 +623,8 @@ def check(cls, remote=''):
 
     def fedora_version(self):
         pkg = self.pkg_by_name("fedora-release") or \
-            self.all_pkgs_by_name_regex("fedora-release-.*")[-1]
+            self.package_manager.all_pkgs_by_name_regex(
+                "fedora-release-.*")[-1]
```

Comment:
How this could even work now..? good catch.

Reply:
I didn't catch it, `pylint` did